src/pages/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './HomePage/HomePage';
+export * from './TabsPage/TabsPage';
```

Comment:
Do not export directly from an import. Import the desired module and then export it separately.